DOC-1727-precautions-before-system-change (#254)
tigergraph/server-docs: modules/cluster-and-ha-management/pages/expand-a-cluster.adoc

:page-aliases: tigergraph-server:cluster-resizing:expand-a-cluster.adoc
:sectnums:

> Expanding a cluster adds more nodes to the cluster, and the data in your cluster is redistributed in the expanded cluster during the expansion. You can also change replication factor during an expansion.

Author: ```suggestion
Expanding a cluster adds more nodes to the cluster.
During the expansion, the cluster's data is redistributed across the expanded cluster.
You can also change replication factor during an expansion.
```
